README.md

```diff
@@ -117,3 +118,132 @@ When you resize the VirtualBox window (e.g. to mazimise it) the display resoluti
 If this doesn't work it may be due to the Guest Additions installed in your VM not matching the version of VirtualBox you have installed.
 The easiest way to fix this is to install the [vagrant-vbguest Vagrant plugin](https://github.com/dotless-de/vagrant-vbguest).
 Note that, if the plugin does update your Guest Additions then you will need to shutdown and restart your VM (using `vagrant up`) in order for them to take effect.
+
+## Amazon AWS
+
+It is possible to run using Vagrant on an Amazon AWS EC2 virtual machine. To do this you will need to install the the [`vagrant-aws`](https://github.com/mitchellh/vagrant-aws) plugin. You do not need VirtualBox. You should ensure that you are using a recent version of Vagrant to enable the AWS plugin to work, and you will first need to run the command
+```
+vagrant box add dummy https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box
+```
+in a different directory to your `metomi-vms` directory.
+
+Some set-up is required within the AWS console. You will first need to:
+
+1. Generate a key to allow you to connect to the VM
+2. Create a security group and restrict incoming access to the IP address of your computer 
+3. Create a user and make a note of the required information 
+
+The information in points 1 & 2 will need to be saved to a file called **_aws-credentials_** - an example one is provided which looks like
+```
+export AWS_KEY='AAAAAAAAAAAAAAAAAAAA'
+export AWS_SECRET='BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB'
+export AWS_KEYNAME='CCCCCCCCC'
+export AWS_KEYPATH='/full/path/to/CCCCCCCCC'
+```
+If you are using Windows you may want to replace the options in the Vagrantfile directly, but be careful not to commit this information back to a public repository.
+
+**Note** that because the default username for EC2 Ubuntu VMs is **_ubuntu_** the `/home/ubuntu` directory has also been symbolically linked to `/home/vagrant`. You should continue to work under the `ubuntu` user as normal.
+
+### VM size
+
+There are many different sizes of VM to choose from (known as [instance types](https://aws.amazon.com/ec2/instance-types/)), some of which will be eligible for the free tier, e.g. `t2.micro` that has 1 CPU and 1GB of memory. To be able to run the UM you will need to select a larger type, such as `t2.medium`(2 CPUs and 4GB of memory) or `t2.large`(2 CPUs and 8GB of memory). This is changed in the `aws.instance_type` setting in the Vagrantfile. You can also select faster hardware, e.g. the `m5` hardware uses faster Intel Xeon processors with a greater network and storage bandwidth, and may give better performance. Larger and faster options are available, but these will all come with an associated cost.
+
+The hard disk size of the VM can be set to 30GB in the Vagrantfile in the `aws.block_device_mapping` setting. This can be changed as required. The default `t2.micro` size is 8GB.
```

Comment:
30GB is now the default?

Reply:
Yes, 30GB is the default. This is consistent with the current VirtualBox/Vagrant VM settings and is still part of the AWS free tier so wouldn't charge. However it does mean that no subsequent volume resizing is necessary via the AWS console.

Reply:
I just meant that I thought it would be clearer to change "can be set" to "is set"

Reply:
I've tweaked the text which hopefully makes this clearer.